generators/init/index.cjs

```diff
@@ -52,6 +51,9 @@ module.exports = class extends MixedChain {
 
     if (this.options.help) return;
 
+    // Application context for templates
+    this.application = {};
```

Comment:
what about creating a dedicated object for this?
`utils/generation/model/contexts/application-context.js`
`application = { baseName: undefined, baseFolder; undefined, ...}`

Reply:
or in each generator if you prefer to keep them loosely coupled

Reply:
We are using javascript not typescript, I don't see the need for this.

Reply:
giving some guidance and a clear contract for anyone which want to know the attributes that are recognized by the generators.
But anyway as you mentioned, javascript is not so pojo oriented...